sites/friday/src/add-services/clickhouse.md

```diff
@@ -0,0 +1,211 @@
+---
+title: "Clickhouse"
+weight: -110
+description: 
+---
+
+Clickhouse is a high-performance column-oriented, distributed, OLAP (Online Analytical Processing) database.</br>
+It is open source and allows you to generate real-time analytical data reports using SQL queries.</br>
+For more information, see the [Clickhouse documentation](https://clickhouse.com/docs).
+
+{{% note %}} 
+
+{{% vendor/name %}} supports Clickhouse with the following limitations:
+
+- High availibility of service isn't supported.
+- You can only configure single-node Clickhouse clusters.
+
+{{% /note %}} 
+
+## Supported versions
+
+You can only specify the major version.
+Each version represents a rolling release of the latest minor version available from the upstream.
+The latest compatible minor version and patches are applied automatically.
```

Comment:
ClickHouse doesn't do semantic versioning, we cannot talk about `major|minor|patch` versions.

23.3.10 version means:
23: year
3: feature release
10: maintenance/bugfix release

from [this](https://kb.altinity.com/altinity-kb-setup-and-maintenance/clickhouse-versions/) documentation


Suggestion:
Each version represents a rolling release of the latest yearly version available from the upstream.
The latest compatible feature and maintenance versions are applied automatically.